docs/tutorials/field_type/3_implement_the_tweet_type_class.md

```diff
@@ -11,7 +11,7 @@ A Type class must implement `eZ\Publish\SPI\FieldType\FieldType` ("Field Type in
 
 All native Field Types also extend the `eZ\Publish\Core\FieldType\FieldType` abstract class that implements this interface and provides implementation facilities through a set of abstract methods of its own. In this case, Type classes implement a mix of methods from the Field Type interface and from the abstract Field Type.
 
-The recommended way to allow the Field Type to be able to generate content name is through implementing the `eZ\Publish\SPI\FieldType\Nameable` interface.
+The recommended way to allow the Field Type to be able to generate content name is through the `FieldType::getName()`.
```

Comment:
In 3.0 there is no other way then `eZ\Publish\Core\FieldType\FieldType::getName`